src/main/java/org/reaktivity/nukleus/tls/internal/stream/ClientStreamFactory.java

```diff
@@ -791,6 +819,9 @@ private void updateNetworkWindow(
         private int applicationWindowFrames;
         private int applicationReplySlot = NO_SLOT;
         private int applicationReplySlotOffset;
+        private boolean networkReplyEnded;
```

Comment:
Suggest setting `networkReplyWindow` to `-1` instead of adding extra `networkReplyEnded` field.